feature/orchestrator/src/test/java/com/simprints/feature/orchestrator/usecases/CreatePersonEventUseCaseTest.kt

```diff
@@ -70,10 +60,7 @@ internal class CreatePersonEventUseCaseTest {
 
     @Test
     fun `Does not create event if no biometric data`() = runTest {
```

Comment:
Do we need more tests to cover the case that is prevented with this PR?

Reply:
The issue is prevented by removing the repo and only using the step result data. I think the current tests cover that; IIRC, there are some extra checks in places where the data is added to step results.